jeffery_russell-mockups/home_page.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+      <link type="text/css" rel="stylesheet" href="base.css">
+      <title>Learning Journal by Jeffery Russell</title>   
+  </head>
+  <body>
+
+    <header>
+        <h1>Learning Journal<h1>
+        <h3>Code 401 - Advanced Software Development in Python</h3>
+        <h3>by Jeffery Ray Russell</h3>
+    </header>
+
+    <main>
+        <h3>Journal Entries</h3>
+        <a href="form_page_new_entry.html">Create New Journal Entry</a>
+        <table id="new_entries_table">
+            <tr>
+                <th>Date</th>
+                <th>Title</th>
+                <th>Edit</th>
+            </tr>
+            <tr>
+                <td>8/08/2016</td>
+                <td><a href="detail_page.html">Title_here</a></td>
+                <td><a href="form_page_existing_entry.html">Edit</a></td>
+            </tr>
+        </table>
```

Comment:
using tables for non-tabular data is kind of frowned upon these days.  The data that will go here is more a list than a table, right?